src/main/java/pl/simpleascoding/tutoringplatform/service/user/UserServiceImpl.java

```diff
@@ -33,6 +35,16 @@ class UserServiceImpl implements UserService {
     private static final String CONFIRM_REGISTER_MAIL_TEXT = "Hi %s, please visit the link below to confirm your email address and activate your account: \n%s";
     private static final String CONFIRM_REGISTER_URL = "%s/confirm-registration?tokenValue=%s";
```

Comment:
Please change it to:
REGISTRATION_LINK

Reply:
Not related to this feature - will be fixed in a separate pull request